src/error.rs

```diff
@@ -14,11 +15,14 @@ impl<T, E: Debug + Display> ErrorExt<T, E> for Result<T, E> {
         }
         Some(self.unwrap())
     }
-    fn unwrap_or_print(self) -> T {
+    fn unwrap_or_print_with_context(self, context: &str) -> T {
         if let Err(error) = &self {
-            eprintln!("{}: {}", "error".red().bold(), error);
+            eprintln!("{}: {}{}", "error".red().bold(), context, error);
```

Comment:
It's probably worth doing

```
if context != "":
    context += " "
```

but I ran out of time for this PR and I know that Rust wouldn't make it easy.